src/rootScanner.ts

```diff
@@ -224,11 +225,13 @@ export default class RootScanner {
     }
 
     private cacheModulesFromMain(moduleName, modulePath, packageJson) {
-        if (!packageJson.hasOwnProperty('main'))
-            return;
-        let mainFilePath = path.join(modulePath, packageJson.main);
+        let mainFilePath = path.join(modulePath, 'src/index.js');
```

Comment:
I think there maybe no `src/index.js` in node_modules in most of case,  `src` directory will ignore in npmignore .

Reply:
It is actually a common pattern with flow types to include the `src` directory when publishing to npm. This is how flow will resolve types in node_modules.